javascript/ql/src/experimental/SockJS/SockJS.qll

```diff
@@ -0,0 +1,47 @@
+/**
+ * Provides classes for working with [SockJS](http://sockjs.org).
+ */
+
+import javascript
+import DataFlow::PathGraph
+
```

Comment:
```suggestion
import javascript

```
You don't need to import `DataFlow::PathGraph` here.  
If you use this library in a path query, that path query should import `DataFlow::PathGraph`.